v2/components/ErrorBoundary.tsx

```diff
@@ -47,9 +48,13 @@ const ErrorMessage: FC<{ className?: string }> = ({ className }) => {
         )}
       >
         <div className="flex justify-center mt-2">
-          <div className="w-3/5 lg:w-1/2 min-h-[200px]">
-            <img src="/animations/failure.gif" />
-          </div>
+          <Image
+            alt="Epic Fail"
+            src="/animations/failure.gif"
+            width={250}
+            height={250}
+            loading="eager"
```

Comment:
This will make the image a blocking resource. I added a workaround in `UserFlowsAnimationsProvider` so the images are ready when they are requested.

Reply:
fixed